proxy/http/HttpConfig.h

```diff
@@ -358,6 +361,38 @@ struct HttpConfigPortRange {
   }
 };
 
+namespace HttpForwarded
+{
+// Options for what parameters will be included in "Forwarded" field header.  Not mutually exclusive, except that there should be
+// only one "by" option enabled.  (I did not use enum class because the enum values would not implicitly convert to integral types).
+//
+namespace Option
+{
+  enum E {
```

Comment:
I think the standard is to use all caps for enum values. In that case, since it's already in a namespace, I'd just have `enum Option` and drop the nested enum `E`.